<!DOCTYPE html>
<html lang="en">
  <!-- Head tag -->
  <head><meta name="generator" content="Hexo 3.8.0">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- Title -->
  
  <title>Supplementary Information and Tools - RNA-Sick</title>

  <!--Favicon-->
  <link rel="icon" href="favicon/favicon.ico">

  <!--Description-->
  
      <meta name="description" content="Supplementary InformationGenome-wide Identification and Analysis of MADS-box Gene Family in Bitter Gourd – Method DetailsThis part is used to document">
  

  <!--Author-->
  
      <meta name="author" content="謝晨 (Chen Hsieh)">
  

  <!-- Pure CSS -->
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
  <link href="https://fonts.googleapis.com/css?family=Crimson+Text|Open+Sans:300,800" rel="stylesheet">

  <!-- Custom CSS -->
  <link rel="stylesheet" href="/css/styles.css">

  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
  <![endif]-->

  <!-- Google Analytics -->
  

</head>


  <body>
  	<div class="container-fluid navbar-container m-sm-5">
      <!-- Header -->
      <nav class="navbar navbar-toggleable-sm navbar-light px-1 py-3 my-3 mb-sm-5">
  <a class="navbar-brand ml-2" href="/">RNA-Sick</a>
  <button class="navbar-toggler navbar-toggler-right py-2" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse text-center" id="navbarCollapse">
    <ul class="navbar-nav ml-auto my-auto">
      
        <li class="nav-item">
          <a class="nav-link" href="/about">About</a>
        </li>
      
        <li class="nav-item">
          <a class="nav-link" href="/contact">Contact</a>
        </li>
      
        <li class="nav-item">
          <a class="nav-link" href="https://www.instagram.com/rna.sick/">Instagram</a>
        </li>
      
        <li class="nav-item">
          <a class="nav-link" href="https://www.linkedin.com/in/chen-hsieh/">LinkedIn</a>
        </li>
      
        <li class="nav-item">
          <a class="nav-link" href="https://www.facebook.com/rna.sick/">Facebook</a>
        </li>
      
    </ul>
    <hr class="hidden-md-up">
  </div>
</nav>


  		<div class="row">
  			<div class="col-12 mb-4">
  <img class="img-fluid project-img" src="/images/SupDetailAndTools.jpg" alt="Supplementary Information and Tools">
</div>
<div class="col-lg-4 col-12 pt-3 px-4 pr-lg-5">
  <h1>Supplementary Information and Tools</h1>
</div>
<div class="col-lg-8 col-12 pt-lg-3 mb-4 pl-lg-5 px-lg-0 px-4 portfolio-content">
  <h1 id="Supplementary-Information"><a href="#Supplementary-Information" class="headerlink" title="Supplementary Information"></a>Supplementary Information</h1><h2 id="Genome-wide-Identification-and-Analysis-of-MADS-box-Gene-Family-in-Bitter-Gourd-–-Method-Details"><a href="#Genome-wide-Identification-and-Analysis-of-MADS-box-Gene-Family-in-Bitter-Gourd-–-Method-Details" class="headerlink" title="Genome-wide Identification and Analysis of MADS-box Gene Family in Bitter Gourd – Method Details"></a>Genome-wide Identification and Analysis of MADS-box Gene Family in Bitter Gourd – Method Details</h2><p>This part is used to document method details in the poster “Genome-wide Identification and Analysis of MADS-box Gene Family in Bitter Gourd” presented at the “2019 Poster Competition of Department of Horticulture and Landscape Architechture, NTU”.</p>
<hr>
<h3 id="Genome-wide-identification-of-MADS-box-genes"><a href="#Genome-wide-identification-of-MADS-box-genes" class="headerlink" title="Genome-wide identification of MADS-box genes"></a>Genome-wide identification of MADS-box genes</h3><p>Based on predicted transcripts from the latest genome assembly of bitter gourd (ASM199503v1, Strain: OHB3-1) and our <em>de novo</em> assembled transcriptome data (Cultivar: ‘Moon Shine’), hidden Markov model (HMM) search was conducted using Pfam profiles including SRF-TF (PF00319) and K-box (PF01486). Transcript variants and isoforms were merged and pointed to corresponding gene entity. Each of the gene entity can be assigned to a gene symbol from representative genome of bitter gourd on NCBI database. The first gene product of each gene symbol was used as representative protein for further classification analysis. Naming of MADS-box genes in bitter gourd were based on the corresponding best hit of MADS-box genes in watermelon (Wang <em>et al.</em>, 2019).</p>
<h3 id="Phylogenetic-analyses-of-MADS-box-genes"><a href="#Phylogenetic-analyses-of-MADS-box-genes" class="headerlink" title="Phylogenetic analyses of MADS-box genes"></a>Phylogenetic analyses of MADS-box genes</h3><p>Multiple sequence alignments were performed between full-length MADS-box protein sequences from bitter gourd, <em>Arabidopsis</em>, and watermelon using MUSCLE (Edgar, 2004) with default settings in Seqotron (Fourment and Holmes, 2016). An unrooted maximum-likelihood (ML) tree was constructed using the MEGA7. The tree nodes were evaluated by bootstrap analysis for 1000 replicates.</p>
<h3 id="Conserved-Motif-analysis"><a href="#Conserved-Motif-analysis" class="headerlink" title="Conserved Motif analysis"></a>Conserved Motif analysis</h3><p>MEME-suite (Bailey <em>et al.</em>, 1994; Bailey <em>et al.</em>, 2009) was applied to predict the conserved motifs of the MADS-box with the following parameter settings: number of repetitions -anr (any number of repetitions), maximum number of motifs -7, minimum motif width -6 and maximum motif width -69.</p>
<h3 id="Treatments-and-growth-conditions-of-plant-materials"><a href="#Treatments-and-growth-conditions-of-plant-materials" class="headerlink" title="Treatments and growth conditions of plant materials"></a>Treatments and growth conditions of plant materials</h3><p> Monoecious cultivar ‘Moon Shine’ (MS) and gynoecious cultivar ‘BM3250’ (BM) were chosen. The ratio of staminate/pistillate flowers of MS is 36.54. All of the plants were raised in growth chamber. Day and night temperature were set as 28 ̊C and 25 ̊C, respectively. Day and night durations were set as 16 hours and 8 hours, respectively. Four sex-determination alteration treatments were appied on MS. Gibberelin treatment (GA) and NAA treatment (NAA) refers to the foliar feeding of 100mg/L GA3 and 25mg/L NAA after raising the seedlings to 6~8-leaf stage and 2-leaf stage, repectively. Foliar feeding were conducted for three times, three days apart for each application. Low- temperature seed-soaking treatment (COLD) and gibberelin and low-temperature combined treatment (GC) refers to the seed soaking under 4 ̊C for 15 days in sdH2O and 100mg/L GA3, respectively. A total of six group of seedlings, two control groups of the two cultivars and four treatment groups of MS, were raised to the same stage of 6~8-leaf stage, and then collected for storage in -80 ̊C for further analysis.</p>
<h3 id="Expression-analysis-of-MADS-box-genes"><a href="#Expression-analysis-of-MADS-box-genes" class="headerlink" title="Expression analysis of MADS-box genes"></a>Expression analysis of MADS-box genes</h3><p>In order to construct a comprehensive gene expression matrix of MADS-box genes in bitter gourd, expression profiles of different organs, cultivars, and treatment response of bitter gourd were obtained by RNA-sequencing results from not only NCBI open data but also our own plant materials. SRA-tool was used to retrieve raw fastq data of PRJNA 213805 and PRJNA 319011 from NCBI BioProjects database. PRJNA213805 is a BioProject from India and contains transcriptomes of gynoecious line (Gy323) and monoecious lines (DRAR1) of bitter gourd. PRJNA 319011 is a BioProject contains transcriptomes from different organs of bitter gourd. Total RNA of plant materials mentioned in the “Treatments and growth conditions of plant materials” section were extracted and sequenced with NGS platform. A total of 12 groups of RNA-sequencing data sets were used to construct a comprehensive bitter gourd expression profile. Draft genome sequence (Urasaki et al., 2017) and its predicted transcriptome was used as reference to construct a comprehensive gene expression matrix with transcript quantification software Kallisto (Bray <em>et al.</em>, 2016). According to the result of flower sex inducing effect by each treatment in MS, we would call the comparison between Control (CK) and GA as GA3 responsive female-inducing (GA(f)) effect, comparison between COLD and GC as GA3 responsive male-inducing (GA(m)) effect, comparison between CK and COLD as low-temperature responsive female-inducing (LT(f)) effect, and comparison between CK and NAA as NAA responsive female-inducing (NAA(f)) effect. Normalized value of TPM and fold change by z score method were used to visualize and cluster the overall expression matrix of MADS-box genes in bitter gourd.</p>
<h3 id="Cis-acting-element-analysis-of-promoter-region"><a href="#Cis-acting-element-analysis-of-promoter-region" class="headerlink" title="Cis-acting element analysis of promoter region"></a>Cis-acting element analysis of promoter region</h3><p>New PLACE database (<a href="https://www.dna.affrc.go.jp/PLACE/?action=newplace" target="_blank" rel="noopener">https://www.dna.affrc.go.jp/PLACE/?action=newplace</a>) was used to predict cis-acting elements in the <em>McMADS</em>s’ promoter (1.5 kb upstream from the predicted transcription start site, TSS). In this study, <em>cis</em>-acting elements related to seven biological related categories of interest were selected. The seven categories are hormone responsive elements, tissue specific elements, circadian responsive elements, cell cycle regulation, stress responsive element, TF regulation, light responsive elements. Category related <em>cis</em>-acting elements were picked up according to the presence of keywords in corresponding publication titles. Visualization of each cis-acting elements was conducted with custom python scripts using seaborn rugplot function.</p>
<h3 id="References"><a href="#References" class="headerlink" title="References:"></a>References:</h3><p>Bailey, T.L., M. Bodén, F.A. Buske, M. Frith, C.E. Grant, L. Clementi, J.Y. Ren, W.W. Li, S.N. William. 2009. MEME SUITE: tools for motif discovery and searching. Nucleic Acids Research. 37:W202-W208.</p>
<p>Bailey, T.L., and C. Elkan. 1994. Fitting a mixture model by expectation maximization to discover motifs in biopolymers. Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology. 28-36.</p>
<p>Bray, N.L., H. Pimentel, P. Melsted, and L. Pachter. 2016. <a href="http://www.nature.com/nbt/journal/v34/n5/full/nbt.3519.html" target="_blank" rel="noopener">Near-optimal probabilistic RNA-seq quantification</a>. Nature Biotechnology 34:525–527</p>
<p>Edgar R.C. 2004. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Research. 32(5) 1792-1797.</p>
<p>Fourment, M., and E.C. Holmes. 2016. Seqotron: a user-friendly sequence editor for Mac OS X. BMC Research Notes. 9(106)</p>
<p>Higo, K., Ugawa, Y., Iwamoto, M. and Korenaga, T. 1999. Plant cis-acting regulatory DNA elements (PLACE) database. Nucleic Acids Research. 2(1):297-300.</p>
<p>Urasaki, N., H. Takagi, S. Natsume, A. Uemura, N. Taniai, N. Miyagi, M. Fukushima, S. Suzuki, K. Tarora, M. Tamaki, M. Sakamoto, R. Terauchi, and H. Matsumura. 2017. Draft genome sequence of bitter gourd (<em>Momordica charantia</em>), a vegetable and medicinal plant in tropical and subtropical regions. DNA Research 24:51-58.</p>
<p>Wang, P., S. Wang, Y. Chen, X. Xu, X. Guang, Y. Zhang. 2019. Genome-wide Analysis of the MADS-Box Gene Family in Watermelon. Computational Biology and Chemistry. 80:341-350.</p>
<h1 id="A-Curated-List-of-Tools"><a href="#A-Curated-List-of-Tools" class="headerlink" title="A Curated List of Tools"></a>A Curated List of Tools</h1><h2 id="1-HRM-analysis-with-PCA-and-clustering"><a href="#1-HRM-analysis-with-PCA-and-clustering" class="headerlink" title="1. HRM analysis with PCA and clustering"></a>1. HRM analysis with PCA and clustering</h2><p>Principal Component Analysis (PCA) of High Resolution Melting (HRM) Analysis for high throughput genotyping</p>
<h3 id="Demostration-on-NextJournal"><a href="#Demostration-on-NextJournal" class="headerlink" title="Demostration on NextJournal"></a>Demostration on NextJournal</h3><p><a href="https://nextjournal.com/RNA-Sick/hrm-analysis-with-pca-and-clustering" target="_blank" rel="noopener">https://nextjournal.com/RNA-Sick/hrm-analysis-with-pca-and-clustering</a></p>
<h3 id="Citation"><a href="#Citation" class="headerlink" title="Citation"></a>Citation</h3><p>Chou, L., S.J. Huang, C. Hsieh, M.T. Lu, C.W. Song, F.C. Hsu. A High Resolution Melting Analysis (HRM)-based Genotyping Toolkit for Chilling Requirement in Peach (<em>Prunus persica</em>). Submitted to International Journal of Molecular Sciences on 11/29/2019.</p>
<h2 id="2-Paper-Digest"><a href="#2-Paper-Digest" class="headerlink" title="2. Paper Digest"></a>2. Paper Digest</h2><p><em>Paper Digest</em> is a solution for researchers to quickly grasp the core ideas of a paper</p>
<p>Paper Digest Official Site: <a href="https://www.paper-digest.com/" target="_blank" rel="noopener">https://www.paper-digest.com/</a></p>
<p>Introduciton of Paper Digest by RNA-Sick: <a href="https://www.facebook.com/rna.sick/posts/2485881328333576" target="_blank" rel="noopener">https://www.facebook.com/rna.sick/posts/2485881328333576</a></p>
<h2 id="3-Sourcetrail"><a href="#3-Sourcetrail" class="headerlink" title="3. Sourcetrail"></a>3. Sourcetrail</h2><p>Sourcetrail provides both overview and details by combining an interactive dependency graph, a concise code view and efficient code search, all built into an easy-to-use cross-platform developer tool.</p>
<p>Sourcetrail Official Site: <a href="https://www.sourcetrail.com/?fbclid=IwAR3rYsMRYcltJT449iBCF1X7RPzRw0b2w-PcolIXa-EWvWO33YkQ--8m8H4" target="_blank" rel="noopener">https://www.sourcetrail.com/</a></p>
<p>Introduciton of Sourcetrail by RNA-Sick: <a href="https://www.facebook.com/rna.sick/posts/2481779772077065" target="_blank" rel="noopener">https://www.facebook.com/rna.sick/posts/2481779772077065</a></p>
<hr>
<p>Photo by <a href="https://unsplash.com/@auxoiux?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText" target="_blank" rel="noopener">CHU TAI</a> on <a href="https://unsplash.com/s/photos/seedling?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText" target="_blank" rel="noopener">Unsplash</a></p>

</div>


      </div>
      
  	</div>

    <!-- After footer scripts -->
    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>

  </body>
</html>
